website/blog/2022-10-09-INSTALING_KUBERNETES_DELEGATE_VIA_MINIKUBE/installing_kubernetes_delegate.md

```diff
@@ -0,0 +1,105 @@
+# Installing Harness Kubernetes(K8s) delegate via MiniKube
+
+## Before we begin :
+Let us understand what do we mean by delegates and why is it needed 
+
+A Harness delegate is a software that you install in your deployment target environment such as local network ,VPC,or cluster and run as a service.The delegate performs all operations including deployment and integration.
+The delegate connects all your artifacts,infrastructure,collaboration,verification and other providers with the Harness Manager.
+
+
+Below we have discussed the detailed explanation of how we can install Kubernetes(K8s) delegate.
+
+## Prerequisites 
+- First check if virtualization is available in your local machine
+
+  ```sysctl -a | grep -E --color 'machdep.cpu.features|VMX'```
+
+-  If you see VMX highlighted in your terminal then VT-x feature is enabled in your machine.
+
+![Terminal](./1.png)
+
+- In windows you can check via running following commands in the command prompt and see if virtualization is available:
+
+    ```systeminfo```
+
+- Install kubectl (kubernetes command line )  
+  * Mac - ```brew install kubectl```
+  * Windows - For windows you can refer to the following docs for detailed instructions regarding installation:
+   [Kubectl-Windows](https://kubernetes.io/docs/tasks/tools/install-kubectl-windows/)
```

Comment:
- Hypervisor technology (VirtualBox, VMWare, etc) is a mandate pre-requisite for Minikube and we have to choose the right one based on the platform we are on.
- Minikube will download the required `kubectl` as part of the installation and configures it.

With that said, all the above info along with the doc links to install, and configure the required hypervisor of our choice is provided in the Minikube documentation itself so it's safe to point to https://minikube.sigs.k8s.io/docs/start/#what-youll-need

Reply:
### Done thanks for the review !!